WcaOnRails/app/views/competitions/_nav.html.erb

```diff
@@ -211,6 +211,14 @@
     </div>
     <div id="competition-data">
       <h3>
+        <% if current_user %>
+          <span class="bookmark-icon" id="not-bookmarked" data-toggle="tooltip" data-placement="bottom" title="<%= I18n.t('competitions.competition_info.bookmark') %>" <% if current_user.is_bookmarked?(@competition) %>style="display: none;"<% end %>>
```

Comment:
I think the way the javascript is written below, there have to be two `span`s for it to work. You could definitely make this work with a single span, but it would involve writing more jquery, which isn't exactly fun :p

Reply:
What Jeremy said.  It's easier in javascript with two spans.

Reply:
Gotcha.